### ISSUE TITLE
Bug: Back button in navigation has weird behavior

### ISSUE THREAD

**Issue by stefanhayden** [ProgressiveCoders/Spoke#2002]
**Describe the bug**
When I am sending first texts, as I start hitting the enter key to send them, the previous arrow becomes active. Clicking that arrow does not bring you to the previous contact. In fact I have no idea who it brings you to. If you wait a few seconds the `operationName: "getContacts"` graphql triggers and will turn the back arrow back to deactivated.

![send_first_texts](https://user-images.githubusercontent.com/87616/119906101-8e753400-bf1b-11eb-9853-797bbdf68904.gif)

**To Reproduce**
1. Go to send new texts
2. Click on send
3. Observe the back arrow is active but does unexpected things
4. feel confused

**Expected behavior**
It would be nice if the back arrow either stayed deactivated or was not visible when sending initial texts.

**Comment by arena** [comment]
Ideally, there would be a very different UI for the sending of initial texts. Currently, it's a distilled version of the responding UI. I really don't think it needs the forward and back buttons at all. It does need some sort of counter, but what you really want to know is how many you've sent and, mostly, how many you still need to send as opposed to which individual message you are on.

I think the easiest fix for the bug in the short term would be to remove the arrows

Long term, it would be great to make the sending UI even easier to use for the repetitive task of sending

**Comment by stefanhayden** [comment]
right. I guess it's just keeping the same interface as regular texting. I think we just need to be clear on when to show them or not show them. I'm all for hiding both when sending first texts.

In other situations I'm less sure of the correct behavior.

**Comment by arena** [comment]
Here is a sketch from an early exploration of the texting UI when I thought that the Sending UI might be in scope for redesign. It's rough and the header is not at all like the header we have now. The goals I had in mind (and still do) would be to offer texters a gigantic target for "Send" and a progress bar showing how far through their current batch they are. 
![image](https://user-images.githubusercontent.com/411070/121819603-d34ed980-cc42-11eb-8727-51cfaa3419fb.png)

**Comment by arena** [comment]
I updated the design here 
https://github.com/MoveOnOrg/Spoke/issues/1577
